Sources/NostrSDK/CustomEmoji.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  CustomEmoji.swift
+//
+//
+//  Created by Terry Yiu on 11/11/23.
+//
+
+import Foundation
+
+/// [NIP-30 Custom Emoji](https://github.com/nostr-protocol/nips/blob/master/30.md)
+public class CustomEmoji: CustomEmojiValidating, Equatable {
+    public static func == (lhs: CustomEmoji, rhs: CustomEmoji) -> Bool {
+        lhs.isEqual(to: rhs)
+    }
+
+    /// A name given for the emoji, which MUST be comprised of only alphanumeric characters and underscores.
+    let shortcode: String
+
+    /// A a URL to the corresponding image file of the emoji.
+    let imageUrl: URL
```

Comment:
Is it `imageUrl` or `imageURL`?

Reply:
I don't really care too much, but my personal way is to do `imageURL` because "URL" is not a word, it's an acronym. When the name of a thing is just "url", I'll do `url`. Doesn't matter to me though.